app/src/main/java/ch/epfl/sdp/game/PreyActivity.kt

```diff
@@ -159,6 +159,8 @@ class PreyActivity : AppCompatActivity(), ILocationListener, GameTimerFragment.G
         EndGameHelper.startEndGameActivity(this, gameData.initialTime, 0)
     }
 
-
+    override fun onBackPressed() {
+        //No code
+    }
```

Comment:
Is this function really usefull?

Reply:
Yes it's to disallow users from pressing the back button to go back to the lobby once the game started, also to avoid going back by mistake

Reply:
ok